README.md

```diff
@@ -110,6 +110,17 @@ their options when creating them. Example:
 L.marker([51.50915, -0.096112], { pmIgnore: true }).addTo(map);
 ```
 
+Enable leaflet-geoman on an ignored layer:
+```js
+layer.options.pmIgnore = false;
+L.PM.reInitLayer(layer);
+```
+
+You can also pass the `optIn` option, so that on a layer `pmIgnore` must set to `false`
```

Comment:
I don't understand this sentence 🤔. What exactly happens here?

Reply:
Ah I understand now. Please change the sentence to this:

When passing `optIn: true`, a layers `pmIgnore` property has to be set to `false` to get initiated.